VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -754,6 +754,13 @@ export abstract class WaitingRoomBaseDirective {
         return this.videoCallService.retrieveJudicialRoom(this.conference.id, this.participant.id);
     }
 
+    isQuickLinkParticipant(): boolean {
```

Comment:
I think this should be isQuickLinkUser or something similar as this will get confusing as to what we mean by participant

Reply:
@FarronF I will and have already refactored that in the story 7905 to be more generic when dealing with QL Participant/Observer